fs/src/commonMain/kotlin/com/suparnatural/core/fs/FileSystem.kt

```diff
@@ -82,6 +87,12 @@ expect object FileSystem {
      */
     fun writeFile(pathComponent: PathComponent, contents: String, create: Boolean = false, encoding: ContentEncoding = ContentEncoding.Utf8): Boolean
 
+    /**
+     * Writes `contents` to the file located at `path`. If `create` is true, then file is created if it does not exist.
+     * Returns true if operation is successful, otherwise false.
+     */
+    fun writeFile(path: String, contents: ByteArray, create: Boolean): Boolean
```

Comment:
Can you add `writeFile(path: PathComponent, contents: ByteArray, create: Boolean): Boolean` to keep it consistent with the rest of the API?